src/assets/scss/_styleguide.scss

```diff
@@ -0,0 +1,26 @@
+   .color-block{
+    @at-root #{&}__name {
```

Comment:
Why `@at-root #{&}...` instead of `&...` ? If this file is included inside a selector, I expect `.color-block` and its elements/modifiers to be declared inside that scope and not at the root.